docs/user_guide/quickstart.rst

```diff
@@ -0,0 +1,168 @@
+Quickstart
+==========
+
+Generate a container
+--------------------
+
+Use the command :code:`neurodocker generate` to generate containers. In the examples below,
+we generate containers with `Nipype <https://nipype.readthedocs.io/en/latest/>`_,
+`Jupyter Notebook <https://jupyter.org/>`_, and `ANTs <https://github.com/ANTsX/ANTs>`_.
+
+Please see the :doc:`examples` page for more.
+
+Docker
+~~~~~~
+
+Run the following code snippet to generate a `Dockerfile <https://docs.docker.com/engine/reference/builder/>`_.
+This is a file that defines how to build a Docker image.
+
+(This requires having `Docker <https://docs.docker.com/get-docker/>`_ installed)
+
+.. code-block:: bash
+
+    neurodocker generate docker --pkg-manager apt \
+        --base-image neurodebian:buster \
+        --ants version=2.3.4 \
+        --miniconda version=latest conda_install="nipype notebook" \
+        --user nonroot
+
+You should see a block of text appear in the console. This is the Dockerfile generated
+by *Neurodocker*. Instructions in the Dockerfile are ordered in the same way as the
+arguments in the command-line. To build a Docker image with this, save the output to a
+file in an empty directory, and build with :code:`docker build`:
+
+.. code-block:: bash
+
+    mkdir docker-example
+    cd docker-example
+    neurodocker generate docker --pkg-manager apt \
+        --base-image neurodebian:buster \
+        --ants version=2.3.4 \
+        --miniconda version=latest conda_install="nipype notebook" \
+        --user nonroot > Dockerfile
+    docker build --tag nipype-ants .
+
+Then, you can start a Jupyter Notebook with the following command. This will mount
```

Comment:
```suggestion
The image :code: `nipype-ants` contains :code: `ANTs` and a Python environment with :code: `Nipype` and :code: `Jupyter Notebook`.
You can start a Jupyter Notebook with the following command. This will mount
```